packages/@react-spectrum/table/test/TableSizing.test.tsx

```diff
@@ -984,7 +984,8 @@ describe('TableViewSizing', function () {
         let resizeMenuItem = tree.getAllByRole('menuitem')[0];
 
         triggerTouch(resizeMenuItem);
-        act(() => {jest.runAllTimers();});
+        act(() => {jest.advanceTimersToNextTimer();});
+        act(() => {jest.advanceTimersToNextTimer();});
```

Comment:
I'm not sure what else was running because the slider doesn't disappear in the browser on my iPhone. It's only an issue in the React 16/17 tests.